uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,41 @@
+#
+# Raw Air Data.
+#
+
+# Note: unused vars should be assigned NaN
```

Comment:
Definitely it can come from a single device.
And usually is, the use of cheap sensors and separation of data is to my opinion an exclusion from usual practice.